lib/fluent/plugin/out_s3.rb

```diff
@@ -147,8 +146,8 @@ def ensure_bucket
     end
 
     def check_apikeys
-      @bucket.empty?
```

Comment:
Oh! But `Aws::S3::Bucket#empty?` does not exist.
How about `Aws::S3::Bucket#objects.limit(1)`?